src/api/Controllers/UserController.cs

```diff
@@ -61,4 +61,15 @@ public async Task<IEnumerable<User>> GetAll()
 
         return users;
     }
+
+    [HttpPost("resetAllSettings")]
+    public ActionResult ResetAllSettings()
+    {
+        context.Database.ExecuteSqlRaw("UPDATE bdms.users SET settings_usr = null;");
+
+        // Reset admin settings to initial state
+        context.Database.ExecuteSqlRaw("UPDATE bdms.users SET settings_usr = '{{\"filter\": {{\"custom\": {{\"borehole_identifier\": true, \"project_name\": true, \"landuse\": true, \"alternate_name\": true, \"canton\": true, \"city\": true}}, \"restriction\": true, \"mapfilter\": true, \"restriction_until\": true, \"extended\": {{ \"original_name\": true, \"method\": true, \"status\": true}}, \"kind\": true, \"elevation_z\": true, \"length\": true, \"drilling_date\": true, \"zoom2selected\": true}}, \"boreholetable\": {{ \"orderby\": \"original_name\", \"direction\": \"ASC\"}}, \"eboreholetable\": {{ \"orderby\": \"creation\", \"direction\": \"DESC\"}}, \"map\": {{ \"explorer\": {{ }}, \"editor\": {{ }} }}, \"appearance\": {{ \"explorer\": 1}}}}' WHERE username = 'admin';");
+
+        return Ok();
+    }
```

Comment:
Q: Damit das in jedem Fall klappt, müssten wir auch die conf, die auf einem Feld der codelist gespeichert  wird  (warum auch immer) zurücksetzen. 
Das hier ist soweit ich gesehen habe der einzige Code auf dem die conf vom client editiert wird:

SELECT * FROM bdms.codelist where conf_cli IS NOT NULL and schema_cli = 'layer_kind'

Das ist der Fall wenn in den Settings die default Felder für die Layeranzeige geändert werden.
Auf allen anderen Codes sind die confs nur Farben/Patterns und bleiben m.E. stabil.

Sollen wir das hier im PR auch erledigen? Oder separat?